go/vt/vtgate/planbuilder/operators/route_planning.go

```diff
@@ -42,35 +41,44 @@ type (
 		left, right semantics.TableSet
 	}
 
-	opCacheMap map[tableSetPair]abstract.PhysicalOperator
+	opCacheMap map[tableSetPair]Operator
 )
 
-func CreatePhysicalOperator(ctx *plancontext.PlanningContext, opTree abstract.LogicalOperator) (abstract.PhysicalOperator, error) {
+func mapToPhys(ctx *plancontext.PlanningContext, ops []Operator) (sources []Operator, err error) {
+	for _, in := range ops {
+		physOp, err := CreatePhysicalOperator(ctx, in)
+		if err != nil {
+			return nil, err
+		}
+		sources = append(sources, physOp)
+	}
+	return
+}
```

Comment:
In my opinion, this name is not very intuitive. It does `CreatePhysicalOperator` for a given slice of `Operators`, what about, for instance: `createMultiplePhysicalOperators` or simply `createPhysicalOperators`

Reply:
sure I can change this. using `map` for a function that maps values from one type to another is a pretty well established name, but we don't have much functional code, so maybe that is why you reacted to it